packages/Molecules/Markup/examplePlugins.js

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
It will be better to add some simple guide/docs here to tell others what this example is.